app/serializers/search_and_compare/course_serializer.rb

```diff
@@ -184,9 +184,9 @@ def fees
         }
       else
         {
-          Uk: course_enrichment.fee_uk_eu,
-          Eu: course_enrichment.fee_uk_eu,
-          International: course_enrichment.fee_international,
+          Uk: course_enrichment.fee_uk_eu.to_i,
+          Eu: course_enrichment.fee_uk_eu.to_i,
+          International: course_enrichment.fee_international.to_i,
```

Comment:
I don't see any tests changed to reflect this, are these covered?

Reply:
:cry: